lblod/app-digitaal-loket
# Bundle new LEKP forms

# Description

DL-5498 & DL-5499

This PR : 
- Removes the `Type Correctie` field from LEKP form melding correctie authentieke bron.
- Adds three new forms `LEKP-rapport - Sloopbeleidsplan`, `LEKP-rapport - Collectieve renovatie` and  `LEKP-rapport - Fietspaden`
- Adds a new codelist for `LEKP-rapport - Collectieve energiebesparende renovatie`

# Type of change


# Related services

- enrich-submission
- mu-migrations

# How to test 

1. Make sure you have the new RC image from https://github.com/lblod/enrich-submission-service/pull/16
2. Load migrations
3. Log as Gemeente

**LEKP form melding correctie authentieke bron**

![Screenshot 2023-12-05 at 12-07-29 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/33d44b2a-12ff-4f6d-b009-0193bad5119f)

**LEKP-rapport - Sloopbeleidsplan**

![Screenshot 2023-12-15 at 12-54-02 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/acfef1c6-3339-4a7a-bcfa-87135767bfc0)


**LEKP-rapport - Collectieve energiebesparende renovatie**

![Screenshot 2023-12-15 at 11-45-27 Loket voor lokale besturen](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/1298ccff-1d56-4a7c-bbee-9b44755592ef)


**LEKP-rapport - Fietspaden**

![Fietspaden-form](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/eb6bcb9c-39ce-493b-b9c1-9ef0bc163feb)


4. Save the forms, it should be saved without errors.
5. Send the forms, it should be consumed by app-toezicht-abb and not by app-public-decisions-database.
a. To consume the form you can follow the docs @ https://github.com/lblod/app-toezicht-abb#trigger-import-export-flow-from-app-digitaal-loket
 
# What to check

- Typos, and form structure  

# Links to other PR's

- https://github.com/lblod/manage-submission-form-tooling/pull/40
- https://github.com/lblod/enrich-submission-service/pull/16
- https://github.com/lblod/app-meldingsplichtige-api/pull/38
- https://github.com/lblod/app-public-decisions-database/pull/21
- https://github.com/lblod/app-worship-decisions-database/pull/53
- https://github.com/lblod/app-toezicht-abb/pull/35

# Notes

Bump enrich-submission service 

- drc restart migrations resource cache enrich-submission
